src/aura/CRLP_FilterGroup/CRLP_FilterGroup.cmp

```diff
@@ -58,6 +58,8 @@
     <aura:attribute name="filterRuleFieldType" type="String" description="Determines type of field" default="text"/>
 
     <!-- private attributes -->
+    <aura:attribute name="deletedRuleList" type="List" description="Filter Rules that were deleted" access="private"/>
+    <aura:attribute name="isPageDirty" type="Boolean" default="false" access="private" description="True when there are changes to be saved"/>
```

Comment:
Does this attribute need to be used for the filter group name/description too? If not, why is it only necessary for the filterRules? Is there a process to reset it at any point, like when the rules are saved and mode changes to view, or if a rule is removed/deleted?